openpype/lib/applications.py

```diff
@@ -1302,10 +1302,18 @@ def _prepare_last_workfile(data, workdir):
     )
     data["start_last_workfile"] = start_last_workfile
 
+    workfile_startup = should_workfile_tool_start(
+        project_name, app.host_name, task_name
+    )
+    data["workfile_startup"] = workfile_startup
+
     # Store boolean as "0"(False) or "1"(True)
     data["env"]["AVALON_OPEN_LAST_WORKFILE"] = (
         str(int(bool(start_last_workfile)))
     )
+    data["env"]["WORKFILE_STARTUP"] = (
```

Comment:
Use better name and prefix if possible (e.g. `OPENPYPE_WORKFILE_TOOL_ON_START`).